src/main/java/chessmaster/commands/RulesCommand.java

```diff
@@ -11,13 +11,13 @@ public class RulesCommand extends Command {
     private static final String[] RULES_STRINGS = {
         "Here are simple chess rules to get you started:",
         "",
```

Comment:
Can consider using the ChessPiece children classes' static variables for the exact representation here as well.